frontend/src/Frontend/UI/Dialogs/Receive.hs

```diff
@@ -1,30 +1,254 @@
-
+{-# LANGUAGE RecursiveDo #-}
```

Comment:
I'm a bit suspicious of why we can't use uiDeploymentSettings here like the signing ui does. But I trust you and we're out of time to dig deeply. Code dupe and cleanup later if possible sounds fine. :slightly_smiling_face:

Reply:
This is a suspicion that I share. The original UX flow did not allow for the deployment process to be used. That still holds but I'm wary of _not_ using `uiDeploymentSettings` for obvious reasons. Definitely something for me to follow up on.